src/main/res/values/strings.xml

```diff
@@ -647,6 +647,10 @@
     <string name="accept_invalid_certificates">Accept invalid certificates</string>
     <string name="switch_account">Switch Profile</string>
     <string name="add_account">Add Profile</string>
+    <string name="profile_tag">Profile Tag</string>
+    <string name="enter_tag">Enter Tag</string>
```

Comment:
for the placeholder, maybe write down some examples?

`eg. Work, Family, ...`

Reply:
the examples are in the explanation text :thinking: